src/foam/java/refinements.js

```diff
@@ -1084,14 +1084,18 @@ foam.CLASS({
 
   properties: [
     ['javaType', 'java.util.List'],
+    ['javaInfoType', 'foam.core.AbstractListPropertyInfo'],
+    ['javaJSONParser', 'new foam.lib.json.ListParser()'],
     ['javaFactory', 'return new java.util.ArrayList();'],
-    ['javaJSONParser', 'new foam.lib.json.ListParser()']
   ],
 
   methods: [
     function createJavaPropertyInfo_(cls) {
       var info = this.SUPER(cls);
 
+      var compare = info.getMethod('compare');
+      compare.body = 'return super.compare(o1, o2);';
```

Comment:
Is this necessary?

Reply:
Yes otherwise it will use the SafetyUtil.compare method which fails for Lists

Reply:
Okay, maybe add a comment to explain that, since it won't be obvious to future developers reading the code.

Reply:
Done